docs/mods/suite/Quests/Developer/Quests/Rewards.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Quest Rewards
+sidebar_position: 3
+---
+
+| Type          | Description                                             |
+|---------------|---------------------------------------------------------|
+| Item          | Give an item as a reward                                |
+| Choice Reward | Allow the player to choose from a selection of rewards  |
+| Random Reward | The player gets a random reward from a set reward table |
+| Loot Reward   | Reward a player with loot from a reward table           |
```

Comment:
Might be worth documenting the distinction between "random" and "loot" reward. They both reward a random item from a reward table - the difference is that if the reward table has a chance to drop nothing, then "loot" rewards can reward nothing, while "random" rewards will always reward an item.